src/StaticATokenLM.sol

```diff
@@ -677,4 +728,18 @@ contract StaticATokenLM is
     if (rounding == Rounding.UP) return shares.rayMulRoundUp(rate());
     return shares.rayMulRoundDown(rate());
   }
+
+  /**
+   * @notice Initializes a new rewardToken
+   * @param reward The reward token to be registered
+   */
+  function _registerRewardToken(address reward) internal {
+    require(_startIndex[reward] == 0, 'REWARD_ALREADY_REGISTERED');
```

Comment:
I would better check the _rewardTokens array, and not revert but skip if I will see that it's already there